nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -1487,6 +1497,22 @@ public Description matchVariable(VariableTree tree, VisitorState state) {
     return Description.NO_MATCH;
   }
 
+  public static boolean hasNestedClass(Types types, Type type) {
+    if (types == null || type == null) {
+      return false;
+    }
```

Comment:
If these null checks are necessary the parameters should be `@Nullable`.  But I don't think they are needed?